.github/workflows/turbo_hierarchy.yml

```diff
@@ -0,0 +1,59 @@
+name: Turborepo Hierarchy
+
+on:
+  push:
+    branches:
+      - "develop"
+      - "devnet"
+      - "testnet"
+      - "mainnet"
+      - "releases/iota-*-release"
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  diff:
```

Comment:
Missing from Confluence, I guess it was on purpose but please add it

Reply:
Added it